homeassistant/components/lektrico/manifest.json

```diff
@@ -6,7 +6,7 @@
   "documentation": "https://www.home-assistant.io/integrations/lektrico",
   "integration_type": "device",
   "iot_class": "local_polling",
-  "requirements": ["lektricowifi==0.0.42"],
```

Comment:
Can we bump this in a separate PR?

Reply:
I created a separate PR here:  https://github.com/home-assistant/core/pull/128979
Thank you.